packages/neon/neon/lib/src/blocs/apps.dart

```diff
@@ -70,6 +65,23 @@ class AppsBloc extends InteractiveBloc implements AppsBlocEvents, AppsBlocStates
     unawaited(refresh());
   }
 
+  /// Determines the id of initial app.
+  ///
+  /// It requires [appImplementations] to have both a value and data.
+  /// Returns null when the server does not any app.
+  String? get _initialAppID {
```

Comment:
```suggestion
  String? _getInitialAppFallback() {
```

I don't like that it has so much logic just for a getter function.
You could also add the initialApp as a paramter that is returned instead of null, that would make it a bit cleaner even.